core/src/test/java/org/ehcache/EhcacheBasicGetAllTest.java

```diff
@@ -1740,28 +2480,167 @@ public void testGetAllStoreSomeMatchCacheAccessExceptionBeforeLoaderAllMatchNone
+  public void testGetAllStoreSomeMatchCacheAccessExceptionAfterLoaderAllMatchNoneFail() throws Exception {
+    final FakeStore fakeStore = new FakeStore(getEntryMap(KEY_SET_A, KEY_SET_B), Collections.singleton("keyA3"));
+    this.store = spy(fakeStore);
+
+    final FakeCacheLoaderWriter fakeLoader = new FakeCacheLoaderWriter(getEntryMap(KEY_SET_A, KEY_SET_C, KEY_SET_D));
+    this.loaderWriter = spy(fakeLoader);
+
+    final Ehcache<String, String> ehcache = this.getEhcache(this.loaderWriter);
+
+    final Set<String> fetchKeys = fanIn(KEY_SET_A, KEY_SET_C, KEY_SET_D);
+    final Map<String, String> actual = ehcache.getAll(fetchKeys);
+
+    final Map<String, String> expected = getEntryMap(fetchKeys);
+    assertThat(actual, equalTo(expected));
+
+    final InOrder ordered = inOrder(this.loaderWriter, this.spiedResilienceStrategy);
+    verify(this.store).bulkComputeIfAbsent(eq(fetchKeys), getAnyIterableFunction());
+    // ResilienceStrategy invoked: no assertion for Store content
+    ordered.verify(this.loaderWriter, atLeast(1)).loadAll(this.loadAllCaptor.capture());
+    assertThat(this.getLoadAllArgs(), equalTo(fetchKeys));
+    ordered.verify(this.spiedResilienceStrategy)
+        .getAllFailure(eq(fetchKeys), eq(expected), any(CacheAccessException.class));
+
+    validateStats(ehcache, EnumSet.noneOf(CacheOperationOutcomes.GetOutcome.class));
+    validateStats(ehcache, EnumSet.noneOf(CacheOperationOutcomes.CacheLoadingOutcome.class));
+  }
+
+  /**
+   * Tests {@link Ehcache#getAll(Set)} for
+   * <ul>
+   *    <li>non-empty request key set</li> 
+   *    <li>all {@link Store} entries match</li>
+   *    <li>no {@code CacheLoaderWriter}</li>
+   * </ul>
+   */
+  @Test
+  public void testGetAllStoreAllMatchNoLoader() throws Exception {
+    final FakeStore fakeStore = new FakeStore(getEntryMap(KEY_SET_A, KEY_SET_B));
+    this.store = spy(fakeStore);
+
+    final Ehcache<String, String> ehcache = this.getEhcache(null);
+
+    final Set<String> fetchKeys = fanIn(KEY_SET_A, KEY_SET_B);
+    final Map<String, String> actual = ehcache.getAll(fetchKeys);
+
+    assertThat(actual, equalTo(getEntryMap(fetchKeys)));
+
+    verify(this.store).bulkComputeIfAbsent(eq(fetchKeys), getAnyIterableFunction());
+    assertThat(fakeStore.getEntryMap(), equalTo(getEntryMap(KEY_SET_A, KEY_SET_B)));
+    verifyZeroInteractions(this.spiedResilienceStrategy);
+
+    validateStats(ehcache, EnumSet.noneOf(CacheOperationOutcomes.GetOutcome.class));
+    validateStats(ehcache, EnumSet.noneOf(CacheOperationOutcomes.CacheLoadingOutcome.class));
+  }
+
+  /**
+   * Tests {@link Ehcache#getAll(Set)} for
+   * <ul>
+   *    <li>non-empty request key set</li> 
+   *    <li>all {@link Store} entries match</li>
+   *    <li>{@link Store#bulkComputeIfAbsent} throws before accessing loader</li>
+   *    <li>no {@code CacheLoaderWriter}</li>
+   * </ul>
+   */
+  @Test
+  public void testGetAllStoreAllMatchCacheAccessExceptionBeforeNoLoader() throws Exception {
```

Comment:
There is problem with git diff. I havent modified existing test cases. sometime I feel svn diff is better than git diff